<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pause</class>
 <widget class="QWidget" name="Pause">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alibaba PuHuiTi</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>QLink</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveGame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alibaba PuHuiTi</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>保存游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadGame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alibaba PuHuiTi</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>载入游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitGame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alibaba PuHuiTi</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
